File: Examples/TS/trees_main.ts
import { Tree, LNode, Leaf } from "./trees";
import * as trees from "./trees";
const t: Tree<number, string> = { kind: "LNode", value: { value: "ok", left: { kind: "Leaf", value: { value: 3 } }, right: { kind: "Leaf", value: { value: 5 } } } }
function size2(t: Tree<number, string>): number {
    return trees.size(t);
}

console.log(size2(t))

File: Examples/PY/trees_main.py
from trees import Tree, LNode, Leaf
import trees
t: Tree[int, str] = LNode(value="ok", left=Leaf(value=3), right=Leaf(value=5))
def size2(t: Tree[int, str]) -> int:
    return trees.size(t)

print(size2(t))
